Optimize SEO for images (alt + title tags) (#932)
zkSNACKs/WasabiDoc: docs/using-wasabi/ChangeCoins.md

Let's assume you want to send 0.5 bitcoin to Alice.
In the `Send` tab you select one or more UTXO, these are the inputs of the transaction.

> For example, let's assume that you choose a <img src="/ShieldRed.png" alt="Wasabi Wallet red shield anonimity set" title="Wasabi Wallet red shield anonimity set" class="shield" /> 1 anonymity set UTXO worth 2 bitcoins.

Reviewer: ```suggestion
For example, let's assume that you choose a <img src="/ShieldRed.png" alt="Wasabi Wallet red shield anonymity set" title="Wasabi Wallet red shield anonymity set" class="shield" /> 1 anonymity set UTXO worth 2 bitcoins.
```

### First round CoinJoin change

> Let's say that you have a KYC <img src="/ShieldRed.png" alt="Wasabi Wallet red shield anonimity set" title="Wasabi Wallet red shield anonimity set" class="shield" /> `anonymity set 1` UTXO, and you register it for a CoinJoin round.

Reviewer: ```suggestion
Let's say that you have a KYC <img src="/ShieldRed.png" alt="Wasabi Wallet red shield anonymity set" title="Wasabi Wallet red shield anonymity set" class="shield" /> `anonymity set 1` UTXO, and you register it for a CoinJoin round.
```

Afterwards, depending on the value of your UTXO in relation to the minimum CoinJoin amount and on the value of UTXOs of the other participants, you will receive two or more UTXOs.

> One of those will be the change leftover from the round, and it will retain the <img src="/ShieldRed.png" alt="Wasabi Wallet red shield anonimity set" title="Wasabi Wallet red shield anonimity set" class="shield" /> `anonymity set 1` designation.
> The remaining UTXO(s) will have either a <img src="/ShieldYellow.png" alt="Wasabi Wallet yellow shield anonimity set" title="Wasabi Wallet yellow shield anonimity set" class="shield" />, <img src="/ShieldGreen.png" alt="Wasabi Wallet green shield anonimity set" title="Wasabi Wallet green shield anonimity set" class="shield" /> or <img src="/ShieldCheckmark.png" alt="Wasabi Wallet green checkmark shield anonimity set" title="Wasabi Wallet green checkmark shield anonimity set" class="shield" /> shield.

Reviewer: ```suggestion
One of those will be the change leftover from the round, and it will retain the <img src="/ShieldRed.png" alt="Wasabi Wallet red shield anonymity set" title="Wasabi Wallet red shield anonymity set" class="shield" /> `anonymity set 1` designation.
The remaining UTXO(s) will have either a <img src="/ShieldYellow.png" alt="Wasabi Wallet yellow shield anonymity set" title="Wasabi Wallet yellow shield anonymity set" class="shield" />, <img src="/ShieldGreen.png" alt="Wasabi Wallet green shield anonymity set" title="Wasabi Wallet green shield anonymity set" class="shield" /> or <img src="/ShieldCheckmark.png" alt="Wasabi Wallet green checkmark shield anonymity set" title="Wasabi Wallet green checkmark shield anonymity set" class="shield" /> shield.
```

You can see the `anonymity set` value of any UTXO by hovering your mouse-pointer over its associated shield.

In this example, let's assume that the current minimum amount that can be registered for a CoinJoin round is 0.1 bitcoin.
You happen to have a UTXO worth 0.105 bitcoin, and you select it to be coinjoined.

After this CoinJoin round has concluded, you will receive 2 separate UTXOs.

> The larger UTXO will (most probably) have a colored shield (<img src="/ShieldYellow.png" alt="Wasabi Wallet yellow shield anonimity set" title="Wasabi Wallet yellow shield anonimity set" class="shield" />, <img src="/ShieldGreen.png" alt="Wasabi Wallet green shield anonimity set" title="Wasabi Wallet green shield anonimity set" class="shield" /> or <img src="/ShieldCheckmark.png" alt="checkmark" class="shield" /> based on your [Privacy Level Settings](/FAQ/FAQ-UseWasabi.md#how-can-i-change-the-anonset-target)) that indicates the level of the anonymity set achieved in that CoinJoin round.

Reviewer: ```suggestion
The larger UTXO will (most probably) have a colored shield (<img src="/ShieldYellow.png" alt="Wasabi Wallet yellow shield anonymity set" title="Wasabi Wallet yellow shield anonymity set" class="shield" />, <img src="/ShieldGreen.png" alt="Wasabi Wallet green shield anonymity set" title="Wasabi Wallet green shield anonymity set" class="shield" /> or <img src="/ShieldCheckmark.png" alt="Wasabi Wallet green checkmark shield anonymity set" title="Wasabi Wallet green checkmark shield anonymity set" class="shield" /> based on your [Privacy Level Settings](/FAQ/FAQ-UseWasabi.md#how-can-i-change-the-anonset-target)) that indicates the level of the anonymity set achieved in that CoinJoin round.
```

> The other, much smaller UTXO will be the amount (change) that was left over, and it will have a <img src="/ShieldRed.png" alt="Wasabi Wallet red shield anonimity set" title="Wasabi Wallet red shield anonimity set" class="shield" /> indicating an anonymity set of `1` as it does not have the equal value denomination of the CoinJoin.

Reviewer: ```suggestion
The other, much smaller UTXO will be the amount (change) that was left over, and it will have a <img src="/ShieldRed.png" alt="Wasabi Wallet red shield anonymity set" title="Wasabi Wallet red shield anonymity set" class="shield" /> indicating an anonymity set of `1` as it does not have the equal value denomination of the CoinJoin.
```

But in most cases, especially when coinjoining large amounts, this link can be done easily.
Thus, it is best to assume that the change coin has no anonymity set.

> The small UTXO that is leftover/change from a CoinJoin round is clearly tied to your KYC input UTXO, but the <img src="/ShieldCheckmark.png" alt="Wasabi Wallet green checkmark shield anonimity set" title="Wasabi Wallet green checkmark shield anonimity set" class="shield" /> UTXO is not.
> If afterwards, you combine the <img src="/ShieldRed.png" alt="Wasabi Wallet red shield anonimity set" title="Wasabi Wallet red shield anonimity set" class="shield" /> UTXO with a <img src="/ShieldCheckmark.png" alt="Wasabi Wallet green checkmark shield anonimity set" title="Wasabi Wallet green checkmark shield anonimity set" class="shield" /> UTXO in a single transaction, then it is clear that both came from the same owner.
> Since your identity was already known in the <img src="/ShieldRed.png" alt="Wasabi Wallet red shield anonimity set" title="Wasabi Wallet red shield anonimity set" class="shield" /> `anonymity set 1`, it can now be linked to your coinjoined UTXO.

Reviewer: ```suggestion
The small UTXO that is leftover/change from a CoinJoin round is clearly tied to your KYC input UTXO, but the <img src="/ShieldCheckmark.png" alt="Wasabi Wallet green checkmark shield anonymity set" title="Wasabi Wallet green checkmark shield anonymity set" class="shield" /> UTXO is not.
If afterwards, you combine the <img src="/ShieldRed.png" alt="Wasabi Wallet red shield anonymity set" title="Wasabi Wallet red shield anonymity set" class="shield" /> UTXO with a <img src="/ShieldCheckmark.png" alt="Wasabi Wallet green checkmark shield anonymity set" title="Wasabi Wallet green checkmark shield anonymity set" class="shield" /> UTXO in a single transaction, then it is clear that both came from the same owner.
Since your identity was already known in the <img src="/ShieldRed.png" alt="Wasabi Wallet red shield anonymity set" title="Wasabi Wallet red shield anonymity set" class="shield" /> `anonymity set 1`, it can now be linked to your coinjoined UTXO.
```

Whenever you combine and send more than one UTXO in a single transaction, the lowest anonymity set among the inputs becomes the overriding anonymity set for all of the UTXOs used in that transaction.

### Second round CoinJoin change

> When you take a <img src="/ShieldCheckmark.png" alt="Wasabi Wallet green checkmark shield anonimity set" title="Wasabi Wallet green checkmark shield anonimity set" class="shield" /> 100 anonset UTXO, and you CoinJoin it again in a new 100 participant round, then you receive one <img src="/ShieldCheckmark.png" alt="Wasabi Wallet green checkmark shield anonimity set" title="Wasabi Wallet green checkmark shield anonimity set" class="shield" /> UTXO with combined anonset of 200, and one <img src="/ShieldCheckmark.png" alt="Wasabi Wallet green checkmark shield anonimity set" title="Wasabi Wallet green checkmark shield anonimity set" class="shield" /> leftover/change UTXO with the same anonset as the input UTXO: 100.

Reviewer: ```suggestion
When you take a <img src="/ShieldCheckmark.png" alt="Wasabi Wallet green checkmark shield anonymity set" title="Wasabi Wallet green checkmark shield anonymity set" class="shield" /> 100 anonymity set UTXO, and you CoinJoin it again in a new 100 participant round, then you receive one <img src="/ShieldCheckmark.png" alt="Wasabi Wallet green checkmark shield anonymity set" title="Wasabi Wallet green checkmark shield anonymity set" class="shield" /> UTXO with combined anonymity set of 200, and one <img src="/ShieldCheckmark.png" alt="Wasabi Wallet green checkmark shield anonymity set" title="Wasabi Wallet green checkmark shield anonymity set" class="shield" /> leftover/change UTXO with the same anonymity set as the input UTXO: 100.
```
